epub33/core/index.html

```diff
@@ -1247,6 +1247,12 @@ <h3>Exempt resources</h3>
 					resources (i.e., the requirement for a fallback for the foreign content document covers any
 					rendering issues within it). As the resource is not referenced from an EPUB content document, it
 					automatically becomes exempt from fallbacks.</p>
+
+				<div class="note">
+					<p>Resources that fall under this latter exemption should be identified in the manifest using the
+							<code>exempt</code> property on their [^item^] entry to avoid their being flagged by EPUB
```

Comment:
This "fall under this latter exemption" is a bit vague. Isn't it better to make it more explicit and refer to exceptions that are not content-specific?